ckan/tests/migration/test_revision_legacy_code.py

```diff
@@ -0,0 +1,287 @@
+# encoding: utf-8
+
+from difflib import unified_diff
+from pprint import pprint, pformat
+
+from ckan import model
+
+import ckan.lib.search as search
+from ckan.lib.dictization.model_save import package_dict_save
+from ckan.lib.create_test_data import CreateTestData
+
+from ckan.migration.revision_legacy_code import package_dictize_with_revisions as package_dictize
+
+
+# tests here have been moved from ckan/tests/legacy/lib/test_dictization.py
+class TestPackageDictizeWithRevisions:
```

Comment:
`...(object):` to make sure we don't get a "classic class"

Reply:
👍 changed